packages/sommelier.rb

```diff
@@ -60,7 +60,7 @@ def self.build
       system "echo 'SOMM=\$(pidof sommelier 2> /dev/null)' >> stopsommelier"
       system "echo 'if [ ! -z \"\$SOMM\" ]; then' >> stopsommelier"
       system "echo '  killall -g sommelier' >> stopsommelier"
-      system "echo '  pkill Xwayland' >> stopsommelier"
+      system "echo '  if pgrep "Xwayland"; then killall Xwayland; fi"
```

Comment:
We'll need to bump the version and remove the old binaries.  Just add another hex character from the git hash on the end to bump the version.

Reply:
Please remove the old binaries and this should be ready.